src/nbl/video/CVulkanPhysicalDevice.cpp

```diff
@@ -1128,10 +1134,9 @@ std::unique_ptr<CVulkanPhysicalDevice> CVulkanPhysicalDevice::create(core::smart
             features.rasterizationOrderDepthAttachmentAccess = rasterizationOrderAttachmentAccessFeatures.rasterizationOrderDepthAttachmentAccess;
             features.rasterizationOrderStencilAttachmentAccess = rasterizationOrderAttachmentAccessFeatures.rasterizationOrderStencilAttachmentAccess;
         }
-#if 0
+
         if (isExtensionSupported(VK_KHR_COOPERATIVE_MATRIX_EXTENSION_NAME))
             features.cooperativeMatrixRobustBufferAccess = cooperativeMatrixFeatures.cooperativeMatrixRobustBufferAccess;
-#endif
 
```

Comment:
special request from yours truly, can you check for `maintenance5` extension or `graphics_pipline_library` extension and return nullptr if neither is available.

If they're available get both their feature and property structs into pNext:
https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_maintenance5.html
https://registry.khronos.org/vulkan/specs/latest/man/html/VK_EXT_graphics_pipeline_library.html#_new_structures

But later when you create Logical Device, enable maintenance5 if available, and only enable graphics pipeline libraries as backup if available.

Reply:
hmm `VkPhysicalDeviceMaintenance5Properties` is declared in `vk1.4`. are we updating vulkan-headers now?

Reply:
you can update Vulkan headers, but for anything that is core in vulkan 1.4 you don't use any vulkan14 struct you need to use the extension struct (we're still running 1.3 and all 1.4 features we get as extensions)